packages/client/src/components/Footer/Footer.style.css

```diff
@@ -50,46 +41,37 @@ footer {
 
 .footer-container {
   width: 1220px;
-  height: 140px;
+  height: 155px;
   background-color: rgba(245, 245, 245, 0.05);
```

Comment:
The background color is not compatible with the Figma design.

Reply:
I changed the background color, Thx